### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by adelajaOlamilekan**
​
## Description
This PR contains a bug fix to the authentication page. This commit resolves the internal server error when a user uses the login feature on the deployed app. I also added authentication to the get all videos endpoint.

​
## Related Issue (Link to linear ticket)

​
## Motivation and Context
This bug fix is needed to ensure the authentication function of the app is working

​
## How Has This Been Tested?
This fix has been tested using the swagger docs using all edge cases of authentication

​
## Screenshots (if appropriate - Postman, etc):
​
Screenshot (1265)
Screenshot (1266)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: app/routes/video_routes.py [hunk 1]
```diff
@@ -138,17 +141,24 @@ def upload_video_blob(
...
     Parameters:
+        request: The request object
         username (str): The username for which to retrieve the videos.
         db (Session): The database session.
     Returns:
         List[Video]: A list of Video objects associated with the given
             username.
     """
+    user_id = request.session.get("username")
+    print(user_id)
```
Comment (zxenonx) [inline app/routes/video_routes.py:157]: let's remove this print statement.

**Review by zxenonx — COMMENTED** [review]
Thanks @adelajaOlamilekan

File: app/routes/video_routes.py (hunk 1 above)
Comment (adelajaOlamilekan) [reply app/routes/video_routes.py:157]: Ok boss.

**Review by zxenonx — APPROVED** [review]
Thanks @adelajaOlamilekan

**Pull request merged**